SwiftUIDZ/View/DetailView.swift

```diff
@@ -142,6 +153,20 @@ struct DetailView: View {
             total: Constants.progressViewTotal
         )
     }
+
+    private var magnification: some Gesture {
```

Comment:
```suggestion
    private var magnificationGesture: some Gesture {
```
Не забываем об окончании в названии